### PR TITLE
DEVEXP-606: Update Fax SDK

### PR DESCRIPTION
**Pull request by asein-sinch**

### CODE REVIEW

File: packages/fax/src/models/v3/enums.ts
Comment (JPPortier) [inline packages/fax/src/models/v3/enums.ts:1]: 👍

File: packages/fax/src/models/v3/enums.ts [hunk 1]
```diff
@@ -21,11 +21,14 @@ export type ErrorType =
   | 'FAX_ERROR'
   | 'FATAL_ERROR'
   | 'GENERAL_ERROR'
-  | 'LINE_ERROR';
+  | 'LINE_ERROR'
+  | string;
 
-export type FaxBase64FileType = 'DOC' | 'DOCX' | 'PDF' | 'TIF' | 'JPG' | 'ODT' | 'TXT' | 'HTML' | 'PNG';
+export type FaxBase64FileType = 'DOCX' | 'PDF' | 'TIF' | 'JPG' | 'TXT' | 'HTML' | 'PNG' | string;
```
Comment (JPPortier) [inline packages/fax/src/models/v3/enums.ts:27]: Is there a ticket created toward Doc to highlight discrepancies between newdocs and oas-documentation ?
This modification is not reflecting spec file from oas-documentation

File: packages/fax/src/models/v3/errors/call-error/call-error.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export interface CallError {
+  /** Type of error for the Call */
+  errorType?: 'CALL_ERROR';
+  /** A developer-facing error message */
+  errorMessage?: string;
+  /** The error code returned describing the error of the call */
+  errorCode?: CallErrorCodeEnum;
+}
+
+export type CallErrorCodeEnum = '11' | '15' | '16' | '17' | '19' | '30' | '32' | '34' | '43' | '49';
```
Comment (JPPortier) [inline packages/fax/src/models/v3/errors/call-error/call-error.ts:13]: Adding `| string` will let list dynamically expandable

File: packages/fax/src/models/v3/errors/document-conversion-error/document-conversion-error.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+export interface DocumentConversionError {
+  /** Type of error for the Document conversion */
+  errorType?: 'DOCUMENT_CONVERSION_ERROR';
+  /** The error message */
+  errorMessage?: string;
+  /** The error code returned during document conversion. */
+  errorCode?: DocumentConversionErrorCodeEnum;
+}
+
+export type DocumentConversionErrorCodeEnum = '4' | '54' | '55' | '57' | '69' | '122' | '128' | '129' | '130' | '133';
```
Comment (JPPortier) [inline packages/fax/src/models/v3/errors/document-conversion-error/document-conversion-error.ts:13]: Adding `| string` will let list dynamically expandable

File: packages/fax/src/models/v3/errors/fax-error/fax-error.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+export interface FaxError {
+  /** Type of error for the fax */
+  errorType?: 'FAX_ERROR';
+  /** A developer-facing error message */
+  errorMessage?: string;
+  /** The error code returned describing the error of the call */
+  errorCode?: FaxErrorCodeEnum;
+}
+
+export type FaxErrorCodeEnum = '6' | '7' | '8' | '10' | '12' | '13' | '14' | '18'
```
Comment (JPPortier) [inline packages/fax/src/models/v3/errors/fax-error/fax-error.ts:13]: Adding `| string` will let list dynamically expandable

File: packages/fax/src/rest/v3/fax-service.ts [hunk 5]
```diff
@@ -10,7 +10,9 @@ import { ServicesApi } from './services';
  *  - emails
  */
 export class FaxService {
-  public readonly emails: EmailsApi;
+  /** @deprecated Use faxToEmail instead */
```
Comment (JPPortier) [inline packages/fax/src/rest/v3/fax-service.ts:13]: 🥇

File: packages/fax/src/models/v3/errors/document-conversion-error/document-conversion-error.ts (hunk 3 above)
Comment (asein-sinch) [reply packages/fax/src/models/v3/errors/document-conversion-error/document-conversion-error.ts]: Fixed

File: packages/fax/src/models/v3/errors/fax-error/fax-error.ts (hunk 4 above)
Comment (asein-sinch) [reply packages/fax/src/models/v3/errors/fax-error/fax-error.ts:13]: Fixed

File: packages/fax/src/models/v3/enums.ts (hunk 1 above)
Comment (asein-sinch) [reply packages/fax/src/models/v3/enums.ts:27]: No, I'll bring it to the agenda of the next meeting with the doc team

**Review by JPPortier — APPROVED** [review]

**Pull request merged**